tests/behat/features/ra_management_grants.feature

```diff
@@ -0,0 +1,181 @@
+Feature: A RA(A) should only have access to certain pages
+
+  Scenario: Provision a institution and a user to promote later on by an authorized institution
+    Given institution "stepup.example.com" can "select_raa" from institution "stepup.example.com"
+    And institution "institution-a.example.com" can "use_ra" from institution "stepup.example.com"
+    And institution "institution-b.example.com" can "use_raa" from institution "stepup.example.com"
+    And institution "institution-d.example.com" can "select_raa" from institution "institution-d.example.com"
```

Comment:
This line can be removed. Also goes for line 8.

Reply:
This is needed to test the routes for a raa from another institution (joe-d-raa).